### PR TITLE
feat: add backupConfigPath method

### PR DESCRIPTION
**Pull request by ONG-YA**
#### Which issue(s) this PR fixes (resolves / be part of):

fixes #

### CODE REVIEW

**Comment by ONG-YA** [comment]
#1511

**Comment by kagaya85** [comment]
Hi @ONG-YA, Thanks for your contribution, is this PR ready for review?

File: contrib/config/apollo/apollo.go [hunk 1]
```diff
@@ -71,6 +72,14 @@ func WithNamespace(name string) Option {
 	}
 }
 
+
+// WithBackupConfigPath with apollo config backupConfigPath
+func WithBackupConfigPath(backupConfigPath string) Option {
```
Comment (tonybase) [inline contrib/config/apollo/apollo.go:77]: WithBackupPath

**Review by shenqidebaozi — APPROVED** [review]
LGTM

**Review by tonybase — APPROVED** [review]

**Pull request merged**